src/plot_api/plot_config.js

```diff
@@ -32,10 +32,11 @@ var configAttributes = {
 
     plotlyServerURL: {
         valType: 'string',
-        dflt: 'https://plot.ly',
+        dflt: '',
         description: [
-            'Sets base URL for the \'Edit in Chart Studio\' (aka sendDataToCloud) mode bar button',
-            'and the showLink/sendData on-graph link'
+            'When set it determines base URL for',
+            'the \'Edit in Chart Studio\' (aka sendDataToCloud) mode bar button',
+            'and the showLink/sendData on-graph link.'
```

Comment:
Can we at least include the url for CSC in the description? I guess that should be `https://chart-studio.plotly.com`? Something like "To enable sending your data to Plotly's public cloud, you need to set both plotlyServerURL to 'https://chart-studio.plotly.com' and showSendToCloud to true"

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/4690/commits/7d5492d07667d163b9cd31a850b97ce61119565b using `https://plotly.com`

Reply:
Extra info added in https://github.com/plotly/plotly.js/pull/4690/commits/9959cb89065cee2c7bcfb07a48372babc6a591a5.

Reply:
Thanks - I'd take `https://plotly.com` out, since I believe that's not going to work, and just include `https://chart-studio.plotly.com`

Reply:
Let's not use `https://plotly.com` as this will confuse people, let's only use `https://chart-studio.plotly.com` as the example.

Let's also not call it "Plotly's public cloud" but rather "Chart Studio Cloud" please.

Finally, let's replace `sendDataToCloud` (which is not one of the options!) with `showEditInChartStudio/showSendToCloud` which are slight variants on each other...

Reply:
Revised in https://github.com/plotly/plotly.js/pull/4690/commits/d81e59829a66796eceefd63153ba0c428255bc74.